plugins/meson.build

```diff
@@ -10,7 +10,7 @@ sources += [
   'plugins/scaleflux/sfx-nvme.c',
   'plugins/seagate/seagate-nvme.c',
   'plugins/shannon/shannon-nvme.c',
-  'plugins/solidigm/solidigm-smart.c',
+  'plugins/solidigm/solidigm-nvme.c',
```

Comment:
If you add a meson.build in the solidigm subdir, just move this also there.

Reply:
My first implementation was how you are suggesting but it had a side effect on the help. The Solidigm plugin would show up as the last in the plug-in list, instead of showing up in alphabetical order.
Are you ok with breaking the alphabetical order pattern?

Reply:
Ah sorry about that. I was under the impression there will be only one file. 

About the ordering: When I added the initial meson.build file I was just to lazy to add to each subdir plugin dir one. I'd like to keep the alphabetical order. I think adding per plugin dir meson.build file would be the cleanest solution.